test/functional/test.run.js

```diff
@@ -288,7 +276,6 @@ describe("jpm run", function () {
       }
 
       it("run with only -v option", function(done) {
-        process.chdir(paramDumpPath);
```

Comment:
The other chdir's that are remaining in this file are useless and I'll remove them. Using the addonDir option is the right way. The tests are running the 'params-dump' addon in the fixtures directory and are testing its output.